README.md

```diff
@@ -27,70 +27,65 @@ In the [`bkc`](https://github.com/intel/ccc-linux-guest-hardening/tree/master/bk
 
 ## Platform Requirements
 
-- The setup requires a Gen-6 or newer Intel CPU (for Intel PT) and sufficient
-  RAM to run several VMs at once.
-- A modified Linux host kernel is used for TDX emulation and VM-based snapshot
-  fuzzing. This setup does not run inside a VM or container!
+- **Intel Skylake or later:** The setup requires a Gen-6 or newer Intel CPU (for
+  Intel PT) and adequate memory (~2GB RAM per CPU, 5-20GB storage per campaign)
 
-## Installation Requirements
+- **Patched Host Kernel:** A modified Linux host kernel is used for TDX emulation
+  and VM-based snapshot fuzzing. This setup does not run inside a VM or container!
 
-The userspace installation and fuzzing workflow has been tested for recent
-Ubuntu (>=20.04) and Debian (>=bullseye). It only requires Python3:
+- **Recent Debian/Ubuntu:** The userspace installation and fuzzing workflow has
+  been tested for recent Ubuntu (>=20.04) and Debian (>=bullseye).
 
-- `python3`
-- `python3-venv`
-
-~~~
-sudo apt-get install python3 python3-venv
-~~~
+- **Know your Kernel:** Working knowledge of Linux console, kernel build and boot,
+  and an idea of the kernel version and feature you want to test.
 
 ## Installation
 
-Clone this repo to a new directory, e.g. `~/tdx`:
-
-```shell
-git clone https://github.com/intel/ccc-linux-guest-hardening ~/tdx
-cd ~/tdx
-```
-
-We use Ansible playbooks to support local or remote installation.
-
-### Local
-
-- installation directory: `<repo_root>/`
+### The installation and the fuzzing runtime requires Python3 and the virtual environment package:
```

Comment:
The sub-heading is too big in my browser..

Reply:
Downgrade the following sub-headings:
- https://github.com/tz0/ccc-linux-guest-hardening/blob/doc-update_main_readme/README.md?plain=1#L45
- https://github.com/tz0/ccc-linux-guest-hardening/blob/62e1c0b39085c9cb1904c0293cc2a3d20ebcdef4/README.md?plain=1#L51
- #### If not yet done, reboot to launch the kAFL/SDV emulation kernel: